api/optimeet/groups/views.py

```diff
@@ -63,6 +63,29 @@ def add_users_to_group(request, group_id):
     
     return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+@csrf_exempt
+@api_view(['GET'])  
+def votes(request, group_id):
+        votes = models.Votes.objects.filter(group_id=group_id)
+        serializer = serializers.VotesSerializer(votes, many=True)
+        return Response(serializer.data)
+    
+@csrf_exempt
+@api_view(['POST'])
+def create_vote(request, group_id):
+
+    try:
+        group = models.Group.objects.get(group_id=group_id)
+    except models.Group.DoesNotExist:
+        raise Http404  
+    
+    request.data['group_id'] = group_id
+    serializer = serializers.VotesSerializer(data=request.data, context={'group_id': group_id})
+    if serializer.is_valid():
+        serializer.save()
```

Comment:
Something I just realized we need to add to this function is updating a vote if one already exists. We should check if the user has already created a vote for this group and if so update that vote instead of creating a new one, otherwise we can create a new one as before.

Reply:
We can get the `user_id` after verifying that the serializer is valid